openlibrary/macros/SearchResultsWork.html

```diff
@@ -85,6 +94,15 @@ <h3 itemprop="name" class="booktitle">
                   </a>
               </span>
       </span>
+      $ is_privileged_user = ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians'))
```

Comment:
Can we check once outside of SearchResultsWork and pass in instead?

Reply:
Done